hasura/metadata/databases/default/tables/public_profiles.yaml

```diff
@@ -213,6 +214,7 @@ update_permissions:
         - avatar
         - background
         - bio
+        - colinks_notification_emails
```

Comment:
shouldn't this be adminOnly update... ?

Reply:
@crabsinger  the mutation is in FronEnd in EditEmailForm.tsx using client not adminClient.
is not it better to change it to an action?

Reply:
https://github.com/coordinape/coordinape/pull/2680/files/b15b03b00bc11b81eab7cf6fdd5c6b89b94fb6bb#diff-7ec456d91c2ea445c54a26ace526b4da9ee2bd8789f6a7de252565995feca557R83
you are right - this is done in FE so client.